packages/gatsby/src/schema/build-connection-fields.js

```diff
@@ -104,7 +104,30 @@ module.exports = type => {
           groupConnections.push(groupConn)
         })
 
-        return groupConnections
+        // make tags arrays case ignored
+        const tags = groupConnections.map(e=>{
```

Comment:
Also, by `extendNode`, do you mean `onCreateNode`? Couldn't find it by searching repo. :D